docs/self-hosted-appcircle/install-server/helm-chart/configuration/docker-k8s-migration.md

```diff
@@ -0,0 +1,822 @@
+  ```
+
+- **Check the Vault data size:**
+  ```bash
+  echo "$APPCIRCLE_DISK_USAGE" | grep "vault_data"
+  ```
+
+You will use those values while configuring the `values.yaml` of the Appcircle server Helm chart.
+
+#### 4. Make Sure the Standalone Appcircle Server Is Running
+
+Before proceeding with the migration, verify that the standalone Appcircle server is operational and healthy. This ensures you can create accurate backups without issues.
+
+Additionally:
+
+- There should be no running builds on the Appcircle during the migration.
+- Prevent external requests while keeping the server healthy by stopping the Nginx service.
+
+To achieve that, you can follow the steps below:
+
+1. **Log in to the standalone Appcircle server:**
+
+2. **Change directory to the `appcircle-server`:**
+
+   ```bash
+   cd appcircle-server
+   ```
+
+3. **Change directory to the `export` directory of the project:**
+   <SpacetechExampleInfo/>
+
+   ```bash
+   cd projects/spacetech/export
+   ```
+
+4. **Stop the Nginx service:**
+   ```bash
+   docker compose stop nginx
+   ```
+
+### Kubernetes Appcircle Server Steps
+
+This section outlines the steps and considerations for deploying the Appcircle server to a Kubernetes cluster. Ensure you review the [Key Considerations](#2-key-considerations) below before proceeding.
+
+#### 1. Prepare for Helm Chart Installation
+
+After reviewing the key considerations, follow the [Appcircle Server Kubernetes Installation Guide](https://docs.appcircle.io/self-hosted-appcircle/install-server/helm-chart/installation/kubernetes) for detailed Appcircle server Helm chart deployment instructions.
+
+#### 2. Key Considerations
+
+- **Stateful Applications:**
+  This guide assumes you are migrating from a standalone Appcircle server to a Kubernetes cluster with stateful applications (e.g., PostgreSQL, MinIO, MongoDB, Vault) managed by the Appcircle Helm chart.
+
+  - If you choose to manage stateful apps **outside the Helm chart's scope**, modify the commands and configurations accordingly.
+  - Refer to the [Production Readiness](https://docs.appcircle.io/self-hosted-appcircle/install-server/helm-chart/configuration/production-readiness) page for additional guidance.
+
+- **Protocol Configuration (HTTP/HTTPS):**
+  Match the protocol used by your standalone Appcircle server:
+
+  - If the standalone server is using `HTTPS`, configure the Helm chart for `HTTPS`.
+  - If the server is running on `HTTP`, adjust the Helm chart configuration for `HTTP`.
+
+- **Storage Allocation:**
+  Update your Helm chart's `values.yaml` file with storage sizes based on the data extracted in the [Check the Data Size](#3-check-the-data-size-on-the-standalone-appcircle-server) section.
+
+  - Refer to the [Storage Configuration](https://docs.appcircle.io/self-hosted-appcircle/install-server/helm-chart/configuration/storage-configuration) page for detailed instructions.
+
+- **Prepare `values.yaml` for Migration:**
```

Comment:
Add disclaimer text that all customized global.yaml settings should be mapped by user to values.yaml by looking at the Kubernetes configuration pages.